### PR TITLE
Fix form

### PR DESCRIPTION
**Pull request by sangeethailango**

### CODE REVIEW

File: lib/country_finder/country_client_behaviour.ex [hunk 1]
```diff
@@ -1,5 +1,5 @@
 defmodule CountryFinder.CountryClientBehaviour do
 
-  @callback all_countries() :: any()
+  @callback all_countries() :: tuple()
```
Comment (nittin-shankar) [inline lib/country_finder/country_client_behaviour.ex]: Good job @sangeethailango. You can be a bit more specific

File: lib/country_finder_web/live/country_live/index.html.heex [hunk 2]
```diff
@@ -1,6 +1,10 @@
 <form class="mt-4" phx-submit="search-country">
-  <div class="flex rounded-md shadow-sm ring-1 ring-inset ring-gray-300 focus-within:ring-2 focus-within:ring-inset focus-within:ring-indigo-600 sm:max-w-md">
-    <.icon name="hero-magnifying-glass" class="flex select-none items-center text-gray-500 sm:text-sm" />
-    <input type="text" name="country-name" id="country-name" class="block flex-1 border-0 bg-transparent py-1.5 pl-1 text-gray-900 placeholder:text-gray-400 focus:ring-0 sm:text-sm sm:leading-6" placeholder="janesmith">
+  <div class="sm:max-w-md">
+   <div class=" px-6 py-1 gap-3 flex rounded-md shadow-md ring-1 ring-inset ring-gray-100 focus-within:ring-2 focus-within:ring-inset focus-within:ring-indigo-600 ">
```
Comment (nittin-shankar) [inline lib/country_finder_web/live/country_live/index.html.heex]:
```suggestion
   <div class="px-6 py-1 gap-3 flex rounded-md shadow-md ring-1 ring-inset ring-gray-100 focus-within:ring-2 focus-within:ring-inset focus-within:ring-indigo-600 ">
```
Remove the unnecessary space

**Review by nittin-shankar — CHANGES_REQUESTED** [review]

**Review by nittin-shankar — CHANGES_REQUESTED** [review]
Run `mix format` before pushing

File: lib/country_finder_web/live/country_live/index.html.heex (hunk 2 above)
Comment (sangeethailango) [reply lib/country_finder_web/live/country_live/index.html.heex]: Done

File: lib/country_finder/country_client_behaviour.ex (hunk 1 above)
Comment (sangeethailango) [reply lib/country_finder/country_client_behaviour.ex]: Done

**Pull request merged**